cmd/install/assets/assets.go

```diff
@@ -15,18 +15,20 @@ import (
 	prometheusoperatorv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
-//go:embed hypershift-operator/*
-//go:embed cluster-api/*
-//go:embed cluster-api-provider-aws/*
-//go:embed cluster-api-provider-ibmcloud/*
-//go:embed cluster-api-provider-kubevirt/*
-//go:embed cluster-api-provider-agent/*
-//go:embed cluster-api-provider-azure/*
-//go:embed cluster-api-provider-openstack/*
-var crds embed.FS
+var crds = []string{
```

Comment:
this would force us include this files within the image file system and anywhere you would want to run the cli install binary.
Let's get back to embed and we can still discriminate at runtime, see func setupCRDs( for some precedents